docs/openapi3.yaml

```diff
@@ -1365,10 +1411,51 @@ components:
         - dsl
         - basic
         example: dsl
+    bench_mark_response:
+      type: object
+
+      properties:
+        test_id:
+          type: string
+          format: uuid
+          example: 4bf5d7ab-f310-4a64-8ec2-d65c06188ec1
+        bench_mark_data:
+          allOf:
+            - $ref: '#/components/schemas/bench_mark_request'
+
+    bench_mark_request:
+      type: object
+      additionalProperties: false
+      properties:
+        errors:
+          type: object
```

Comment:
maybe add here examples for the errors and codes
see here:
maybe we can get free validation of the schema:
https://swagger.io/docs/specification/data-models/dictionaries/

Reply:
fixed